ProcessMaker/Http/Controllers/Api/CssOverrideController.php

```diff
@@ -130,7 +144,10 @@ private function setLoginFooter(Request $request)
         $response = [];
 
         if ((!$setting->wasRecentlyCreated && $setting->wasChanged()) || $setting->wasRecentlyCreated) {
-            $response = ['html' => $setting->getAttributes()['config']['html']];
+            $response = [
+                'changes' => ['loginFooter' => $setting->getAttributes()['config']['html']],
```

Comment:
Please validate if ['html'] exist the use
$setting->getAttribute('config') insteand of $setting->getAttributes()['config']